packages/building-utils/get-workbox-config.js

```diff
@@ -5,6 +5,7 @@ const path = require('path');
 module.exports = function getWorkboxConfig(outputDir) {
   const workboxConfigPath = path.join(process.cwd(), 'workbox-config.js');
   const defaultWorboxConfig = {
+    globIgnores: ['/polyfills/*.js', '/legacy/*.js'],
```

Comment:
I think we should cache polyfills, some of them will be loaded on modern browsers too. What do you think?